addons-l10n/en/editor-devtools.json

```diff
@@ -41,7 +41,7 @@
   "editor-devtools/replace": "Griffpatch: Switch all {name} in this sprite for the variable named:",
   "editor-devtools/top": "send to top",
   "editor-devtools/bottom": "send to bottom",
-  "editor-devtools/clean-plus": "Clean Up Blocks (+)",
+  "editor-devtools/clean-plus": "Clean up blocks +",
```

Comment:
Make the "b" in "blocks" capital to stay inline with scratch 
![image](https://user-images.githubusercontent.com/72760579/114611065-9da36980-9c6e-11eb-9044-a7ede88f02aa.png)

Reply:
Addressed thanks